manim/animation/movement.py

```diff
@@ -131,7 +131,7 @@ def construct(self):
     def __init__(
         self,
         mobject: "Mobject",
-        path: np.ndarray,
+        path: VMobject,
```

Comment:
Either this is rewritten as a string, or a separate import to VMobject is added at the top of the file. 

```suggestion
        path: "VMobject",
```

Reply:
Ah! I see

How dod you figure?

Reply:
It is not a particularly uncommon mistake when working with type hints. The problem has to be with the line you changed, and you can see that the pipeline complains about not knowing `VMobject`: https://github.com/ManimCommunity/manim/runs/4548003604?check_suite_focus=true

Reply:
Aha! I see. Thanks for clearing!